pkgs/development/compilers/mlkit/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, mlton }:
+
+stdenv.mkDerivation rec {
+  pname = "mlkit";
+  version = "4.5.0";
+
+  src = fetchFromGitHub {
+    owner = "melsman";
+    repo = "mlkit";
+    rev = "v${version}";
+    sha256 = "0fc0y40qphn02857fv2dvhwzzsvgixzchx9i6i0x80xfv7z68fbh";
+  };
+
+  nativeBuildInputs = [ autoreconfHook mlton ];
+
+  # Parallel builds fail because of incorrect dependency information
+  # in the Makefile.
+  enableParallelBuilding = false;
+
+  buildFlags = ''mlkit mlkit_libs'';
```

Comment:
https://nixos.org/nixpkgs/manual/#build-phase

makeFlags is supposed to be a list of strings